web/src/components/common/ListingDescription.tsx

```diff
@@ -61,17 +62,23 @@ const ListingDescription: React.FC<ListingDescriptionProps> = ({
     numCommits,
     minCommits,
   },
+  priceFontSize,
+  commitProgressFontSize,
 }) => (
   <Container>
     <DeadlineTag deadline={deadline} largerFont />
     <ListingName>{name}</ListingName>
-    <ListingPrice price={price} oldPrice={oldPrice} largerFont />
+    <ListingPrice
+      price={price}
+      oldPrice={oldPrice}
+      largerFont={priceFontSize === 'large'}
```

Comment:
Can we do the same for ListingPrice's font size as what we do with the CommitProgress? That is, instead of a boolean variable of "largerFont", just use a FontSize enum (can share the same enum or create its own size enum).

Reply:
@xlai20 Okay!